src/components/action-bar/examples/action-bar-floating-with-expand-button.tsx

```
@@ -0,0 +1,64 @@
+import { Component, h, State } from '@stencil/core';
```

ESLint

[error] Remove this unused import of 'State'.

(sonarjs/unused-import)